test/api/dkim-test.js

```diff
@@ -16,6 +16,22 @@ describe('API DKIM', function () {
 
     this.timeout(10000); // eslint-disable-line no-invalid-this
 
+    it('should POST /dkim expect success / key empty', async () => {
```

Comment:
This test would have passed even without the changes. Maybe check that `dnsTxt.value` is set and it does not contain any undefined values?